js/ui/control/attribution.js

```diff
@@ -67,8 +67,14 @@ AttributionControl.prototype = util.inherit(Control, {
             }
             return true;
         });
-
         this._container.innerHTML = attributions.join(' | ');
+
+        var attributionNodes = this._container.childNodes;
+        for (var i = 0; i < attributionNodes.length; i++) {
+            if (attributionNodes[i].classList && attributionNodes[i].classList.contains('mapbox-improve-map')) {
+                this._editLink = attributionNodes[i];
+            }
+        }
```

Comment:
Why did you choose to put this logic within `_updateAttributions` rather than `_updateEditLink`? Did you consider using [`Element#getElementsByClassName`](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByClassName) rather than a manual search for `mapbox-improve-map`?